tests/jballerina-unit-test/src/test/resources/test-src/object/object_initializer_negative.bal

```diff
@@ -92,3 +94,17 @@ type Person4 object {
         return e;
     }
 };
+
+type Too object {
+    public function __init() {
+        io:println("__init");
+    }
+    function name() {
+        self.__init(); // valid
```

Comment:
Shall we add a positive test for this? To check if the `__init()` is actually invoked.

Reply:
Positive test added.